hermes-consumers/src/test/java/pl/allegro/tech/hermes/consumers/consumer/offset/OffsetCommitQueueTest.java

```diff
@@ -155,8 +155,8 @@ public void shouldSetLastToCommitToNullWhenAllMessagesAreDelivered() {
 
     private void addOffsets(long ... offsets) {
         for (Long offset : offsets) {
-            queue.put(new Message("id", "topic", new byte[0], Topic.ContentType.JSON, 12091212L, 120912123L,
-                    new PartitionOffset(KafkaTopicName.valueOf("kafka_topic"), offset, 0)).getOffset());
+            queue.put(new Message("id", "topic", "traceId", new byte[0], Topic.ContentType.JSON,
```

Comment:
Why do we bother with creation of the messages if only what is done here is adding the offset value which looks like will be the same as loop offset variable.